game/scripts/npc/abilities/axe_culling_blade.txt

```diff
@@ -46,7 +46,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "kill_threshold"                                  "350 550 750 1050 1450" //OAA
+        "kill_threshold"                                  "250 450 650 1050 1450" //OAA
```

Comment:
I used your formula for ult scaling. lvl 1 the same as lvl 1 vanilla, lvl 2 is lvl 3 vanilla. The rest you know. Pigeon and I agreed that lvl 5 should be 2250, because you get it at lvl 49.